source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -208,7 +208,7 @@ void BioXASSideBeamline::setupComponents()
 
 	// Endstation safety shutter.
 
-	endstationShutter_ = new  BioXASEndstationShutter("BioXASSideEndstationShutter", "SSH1607-5-I22-01", this);
+	endstationShutter_ = new  CLSExclusiveStatesControl("SSH1607-5-I22-01", "SSH1607-5-I22-01:state", "SSH1607-5-I22-01:opr:open", "SSH1607-5-I22-01:opr:close", this);
```

Comment:
Not quite, @XuetaoLiu-CLS.  Unlike some of the other beamlines, the shutters in BioXASShutters have a component safety aspect compared to the scientific meaning of the fast shutter.  Therefore, these are kept separate to keep their meaning distinct rather grouping them based purely on their name.